keyboards/tada68/keymaps/abishalom/config.h

```diff
@@ -0,0 +1,3 @@
+#include "../../config.h"
```

Comment:
This line should be `#pragma once`.

```suggestion
#pragma once
```